jupyter_ascending/logger.py

```diff
@@ -4,55 +4,22 @@
 
 from loguru import logger
 
-from jupyter_ascending._environment import IS_LOGGING_ENABLED
 from jupyter_ascending._environment import LOG_LEVEL
 from jupyter_ascending._environment import SHOW_TO_STDOUT
 
-__all__ = ["J_LOGGER"]
 
-log_file = os.path.join(tempfile.gettempdir(), "jupyter_ascending", "log.log")
-if IS_LOGGING_ENABLED:
+def setup_logger():
```

Comment:
What was the reason for moving all the setup code into a function? Seems like it should just be set up by default so we don't need to add `setup_logger` in each script we run

Reply:
this new way is better. import side effects are bad. and zack says this works, while my other code didnt, so, great :)